app/src/main/java/com/github/dfa/diaspora_android/activity/SettingsActivity.java

```diff
@@ -30,23 +32,35 @@
 import android.preference.PreferenceScreen;
 import android.support.v7.app.ActionBar;
 import android.support.v7.app.AppCompatActivity;
+import android.view.LayoutInflater;
 import android.view.MenuItem;
+import android.view.View;
+import android.view.Window;
+import android.widget.FrameLayout;
+import android.widget.TextView;
 
 import com.github.dfa.diaspora_android.App;
 import com.github.dfa.diaspora_android.R;
 import com.github.dfa.diaspora_android.data.AppSettings;
+import com.github.dfa.diaspora_android.ui.theming.ColorPalette;
 import com.github.dfa.diaspora_android.util.AppLog;
+import com.github.dfa.diaspora_android.util.ThemeHelper;
+
+import uz.shift.colorpicker.LineColorPicker;
+import uz.shift.colorpicker.OnColorChangedListener;
 
 /**
  * @author vanitas
  */
-public class SettingsActivity extends AppCompatActivity {
+public class SettingsActivity extends ThemedActivity {
     private boolean activityRestartRequired;
+    private ActionBar toolbar;
```

Comment:
On my tablet (4.4), it looks just like on any other device. The only slight problem is, that the actionbar does not get recolored there immediately.